pkgs/development/tools/gomarkdoc/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, buildGoModule, fetchFromGitHub, testers, gomarkdoc }:
```

```suggestion
{ lib
, buildGoModule
, fetchFromGitHub
, testers
, gomarkdoc
}:
```
makes future diffs easier to read